src/components/AppInfo/AppInfo.jsx

```diff
@@ -39,17 +64,20 @@ const AppInfo = () => {
                 <h4>Basic Information</h4>
               </Col>
               <Col>
-                {/*TODO get from backend basic info*/}
                 <FormGroup>
                   <FormLabel>Name</FormLabel>
-                  <FormControl type={'text'} />
+                  <FormControl
+                    type={'text'}
+                    readOnly
+                    defaultValue={current && current.name}
+                  />
```

Comment:
why name field is read only? we can change this value, only appId is unique identificator